### PR TITLE
[W5.4][T10-A2]TravisPhey

### PR DESCRIPTION
**Pull request by TravisPhey**
Made edits to expected.txt

### CODE REVIEW

**Comment by TravisPhey** [comment]
We share the same name pls

**Comment by TravisPhey** [comment]
FINALLY! IT WORKS

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -85,6 +85,14 @@ private void runCommandLoopUntilExitCommand() {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
             CommandResult result = executeCommand(command);
+
+            if(userCommandText.contains("add")) {
+                                Command subCommand = new Parser().parseCommand("list");
+                                CommandResult listAfterAdd = executeCommand(subCommand);
+                                recordResult(listAfterAdd);
+                                ui.showResultToUser(listAfterAdd);
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/Main.java]: Indentation issue.

File: src/seedu/addressbook/commands/AddCommand.java [hunk 2]
```diff
@@ -42,9 +43,9 @@ public AddCommand(String name,
                       String email, boolean isEmailPrivate,
                       String address, boolean isAddressPrivate,
                       Set<String> tags) throws IllegalValueException {
-        final Set<Tag> tagSet = new HashSet<>();
+        final Set<Tag> tagSet = new HashSet<>(); //hashmap
         for (String tagName : tags) {
-            tagSet.add(new Tag(tagName));
+            tagSet.add(new Tag(tagName)); //add tagname to hashmap tagSet
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/AddCommand.java]: Unnecessary comment.

File: src/seedu/addressbook/commands/AddCommand.java [hunk 3]
```diff
@@ -68,9 +69,19 @@ public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
             return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            /*List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);*/
+            //getList();  why cannot resolve?
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/AddCommand.java]: Since we use version control system, you can remove unused code.

**Comment by jeffryhartanto** [comment]
@TravisPhey Some comments added. Please close the PR after reading comments.

Good job for adding a new feature, updating the user guide and I/O test and passing Travis check. However, you haven't updated the JUnit, which is one of the criteria for getting the 5 participation marks. 

Check this [link](https://github.com/nus-cs2103-AY1718S1/forum/issues/41) on how to get back the participation marks.

**Pull request closed without merging**